### PR TITLE
fix: (TNLT-6613) fix all interactives widget not showing

### PR DESCRIPTION
**Pull request by kamiranoff**
- https://nidigitalsolutions.jira.com/browse/TNLT-6613
- Sorry my last fix created an issue where all widgets stopped displaying.

### CODE REVIEW

File: packages/interactive-wrapper/src/interactive-wrapper.android.js [hunk 1]
```diff
@@ -71,10 +69,10 @@ class InteractiveWrapper extends Component {
       config: { dev, environment, platform, version },
       id,
     } = this.props;
-    const { height, hasError, isLoading } = this.state;
+    const { height, hasError } = this.state;
     const uri = `${editorialLambdaProtocol}${editorialLambdaOrigin}/${editorialLambdaSlug}/${id}?dev=${dev}&env=${environment}&platform=${platform}&version=${version}`;
 
-    if (hasError || isLoading) return null;
+    if (hasError) return null;
```
Comment (kamiranoff) [inline packages/interactive-wrapper/src/interactive-wrapper.android.js:75]: setting `isLoading: true` caused this to always render null initially and `onLoadEnd ` was never called.

Comment (c-j-j) [reply packages/interactive-wrapper/src/interactive-wrapper.android.js:75]: @kamiranoff what is displayed whilst the interactive is loading?

**Review by aronshamash — APPROVED** [review]

File: packages/interactive-wrapper/src/interactive-wrapper.android.js (hunk 1 above)
Comment (kamiranoff) [reply packages/interactive-wrapper/src/interactive-wrapper.android.js:75]: 133744fc-89fc-43c0-8535-e70844d04e55
That would be a white box.

Comment (kamiranoff) [reply packages/interactive-wrapper/src/interactive-wrapper.android.js:75]: And I believe then some widgets then have their own loading mechanisms.

**Review by Geoff-Ford — APPROVED** [review]

File: packages/interactive-wrapper/src/interactive-wrapper.android.js (hunk 1 above)
Comment (kamiranoff) [reply packages/interactive-wrapper/src/interactive-wrapper.android.js:75]: @c-j-j I will ask my PM if they want to add a new ticket in the backlog to improve the component.

**Review by c-j-j — APPROVED** [review]

**Pull request merged**